CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # UNRELEASED
 
+**BREAKING CHANGES**
+
+-   `EncryptionSettings.onlyAllowTrustedDevices` has been replaced with
+    `EncryptionSettings.sharingStrategy`, which adds the ability to share only
+    with cross-signed devices.
+
+-   Update matrix-rust-sdk to `11cbf849c`, which includes:
+
+    -   refactor(sdk-crypto): Room key sharing, introduce extensible strategy ([#3605](https://github.com/matrix-org/matrix-rust-sdk/pull/3605))
```

Comment:
Ideally we'd include everything here which affects the behaviour of the library. That should just be a matter of diffing the changelogs in the rust-sdk, but people are terrible at remembering to update them.

```suggestion
**Other changes**

-   Update matrix-rust-sdk to `11cbf849c`, which includes:

    -   refactor(sdk-crypto): Room key sharing, introduce extensible strategy
        ([#3605](https://github.com/matrix-org/matrix-rust-sdk/pull/3605))
    
    -   Log the content of received `m.room_key.withheld` to-device events.
        ([#3591](https://github.com/matrix-org/matrix-rust-sdk/pull/3591))

    -   Attempt to decrypt bundled events (reactions and the latest thread reply) if they are found in the unsigned part of an event.
        ([#3468](https://github.com/matrix-org/matrix-rust-sdk/pull/3468))

```